xml/tuning_tuned.xml

```diff
@@ -0,0 +1,2501 @@
+          </para>
+<screen>&prompt.sudo;<command>tuned-adm off</command></screen>
+        </step>
+        <step>
+          <para>
+            If the status of the &tuned; daemon is <emphasis>active</emphasis>, stop the daemon:
+          </para>
+<screen>&prompt.sudo;<command>systemctl stop tuned</command></screen>
+        </step>
+        <step>
+          <para>
+            To stop the &tuned; daemon from being automatically activated during the next boot,
+            disable the associated systemd service:
+          </para>
+<screen>&prompt.sudo;<command>systemctl disable tuned</command></screen>
+          <para>
+            <emphasis>ALTERNATIVELY</emphasis>, you can simultaneously disable the &tuned; systemd
+            service and stop the &tuned; daemon immediately:
+          </para>
+<screen>&prompt.sudo;<command>systemctl disable --now tuned</command></screen>
+        </step>
+      </procedure>
+    </sect2>
+  </sect1>
+  <sect1 xml:id="sec-tuning-tuned-profiles">
+    <title>&tunedapp; profiles</title>
+
+    <para>
+      &tunedapp; optimizes &sles; systems using predefined profiles with settings tailored for
+      different use cases. These profiles adjust several system settings, including CPU governor
+      policies, disk I/O scheduling, network parameters, and kernel parameters, to enhance
+      performance, energy efficiency, or other system characteristics. Each profile is designed for
+      specific scenarios, such as high throughput, low latency, power saving, or virtualized
+      environments. System administrators can switch between profiles using the
+      <command>tuned-adm</command> command, and customize or combine profiles to meet unique
+      requirements. This section covers the basics of managing, creating and customizing &tunedapp;
+      profiles.
+    </para>
+
+    <sect2 xml:id="sec-tuning-supported-tuned-profiles">
+      <title>Supported &tunedapp; profiles</title>
+      <para>
+        &sles; supports the following profiles:
+      </para>
+      <variablelist>
+        <varlistentry>
+          <term>balanced</term>
+          <listitem>
+            <para>
+              The <literal>balanced</literal> profile provides a general-purpose optimization of
+              the system, offering a good compromise between performance and energy consumption. It
+              dynamically adjusts CPU frequency and power states, and balances I/O and network
+              performance with power saving, making it suitable for most desktop and server
+              environments.
+            </para>
+          </listitem>
+        </varlistentry>
+        <varlistentry>
+          <term>cpu-partitioning</term>
+          <listitem>
+            <para>
+              The <literal>cpu-partitioning</literal> profile is designed for systems where CPU
+              resources need to be partitioned for specific tasks or applications. It configures
```

Comment:
I think repeating the profile name is kind of unnecessary as it's already printed in the `<term>`. You could shorten it like this:

```suggestion
              Designed for systems where CPU
              resources need to be partitioned for specific tasks or applications. It configures
```

If you want to go this path, make the other first sentences in this variablelist consistent.